CONTRIBUTING.md

```diff
@@ -1,108 +1,117 @@
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution;
+this simply gives us permission to use and redistribute your contributions as
+part of the project. Head over to <https://cla.developers.google.com/> to see your current agreements on file or to sign a new one.
 
- You generally only need to submit a CLA once, so if you've already submitted one
- (even if it was for a different project), you probably don't need to do it
- again.
+You generally only need to submit a CLA once, so if you've already submitted one
+(even if it was for a different project), you probably don't need to do it
+again.
 
 ## Code reviews
 
- All submissions, including submissions by project members, require review. We use GitHub pull requests for this purpose. Consult[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more information on using pull requests.
+All submissions, including submissions by project members, require review. We use GitHub pull requests for this purpose. Consult[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more information on using pull requests.
 
 ## Community Guidelines
 
- This project follows[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+This project follows[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
 
 ## Code of conduct
- Before you contribute, please review and agree to abide by our community [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+Before you contribute, please review and agree to abide by our community [Code of Conduct](/CODE_OF_CONDUCT.md).
 
 ## Ways You Can Contribute
-  If you wish to contribute to this project, follow these steps below:
- - Step 1: [Report Issues](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md#step-1-report-issues)
- - Step 2: [Submit a Pull Request](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md#step-2-submit-pull-requests)
- - Step 3: [Fix or Improve Documentation](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md#step-3-fix-or-improve-documentation) 
- - Step 4: [Join Our Community Meetings](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md#step-4-join-our-community-meetings) 
+
+If you wish to contribute to this project, follow these steps below:
+
+- Step 1: [Report Issues](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md#step-1-report-issues)
+- Step 2: [Submit a Pull Request](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md#step-2-submit-pull-requests)
+- Step 3: [Fix or Improve Documentation](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md#step-3-fix-or-improve-documentation)
+- Step 4: [Join Our Community Meetings](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md#step-4-join-our-community-meetings)
 
 ### Step 1: Report issues
```

Comment:
Same comment - are these steps to be followed in a sequence of simply ways to engage / contribute?